samples/dependency-injection/externally-managed-mode/Core_8/Sample/Sample.csproj

```diff
@@ -2,7 +2,6 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.1;netcoreapp2.1;net48</TargetFrameworks>
     <OutputType>Exe</OutputType>
-    <LangVersion>7.3</LangVersion>
```

Comment:
As per https://github.com/Particular/RepoStandards/pull/26, we no longer set the language version.

Reply:
Hmm... this fails the build. Something feels off with that, but I'll follow up separately. I'll revert this change.